components/dash-core-components/src/fragments/Dropdown.react.js

```diff
@@ -155,6 +157,7 @@ const Dropdown = props => {
                 filterOptions={filterOptions}
                 options={sanitizedOptions}
                 value={value}
+                closeOnSelect={multi ? close_on_select : true}
```

Comment:
> On second thought, it would make more sense to not write over the `close_on_select` prop. If someone were to change multi, the prop would be lost and it would always close.

:+1: